src/components/button/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './styles.module.css';
+
+const Button = React.memo(props => {
+    const { children, Style, onClick } = props;
+
+    return <button style={Style} onClick={onClick}>{children}</button>;
```

Comment:
shouldn't we have some default styles too?

Reply:
Done
css file has some styles , rest I will be do by making another branch to update button styles as discussed.